### DISCUSSION TITLE
crash upon note deletion editor

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#18558] [Q&A] [answered]
[logs.zip](https://github.com/ppy/osu/files/8836120/logs.zip)

when i delete a note in the editor my editor crashed. I waited until it happened again to report it to see if it was a fluke, but it has happened twice now.

also, I am now unable to delete notes in the timeline of the editor. Like, right clicking on a note in the timeline does nothing, doesn't show the dropdown.

**Comment by frenzibyte** [comment, marked as answer]
Logs show nothing, but from looking at sentry, you have hit https://github.com/ppy/osu/issues/18077 ([report](https://sentry.ppy.sh/organizations/ppy/issues/1160)).

**Reply by Axilotl17** [reply]
i started my game after to open osu folder.. does that override? 

also, is sentry what happens when it says "this has been automatically reported to the developers" or whatnot (it showed that message then crashed ahha)

**Reply by frenzibyte** [reply]
> i started my game after to open osu folder.. does that override?

Yes, that overwrites the game files, and is mentioned in the bug report form you see when submitting an issue thread.

> also, is sentry what happens when it says "this has been automatically reported to the developers"

Correct.
